docs/nomenclature.md

```diff
@@ -0,0 +1,139 @@
+# Evolution Platform Nomenclature
+
+## Interview and Questionnaire
+
+* **Interview** All data related with an interview goes the interview. It includes the Response (see below) and all metadata (interview id, language, validity, etc.).
+
+* **Questionnaire** The configuration of widgets in each section
+
+* **Widget** Any block of text, image, question or button appearing in the questionnaire, whether or not it would be associated with an Answer
+
+* **Section** Part of the questionnaire having its own page during the interview
+
+* **Answer** Answer to a specific question, field or attribute during the interview. An Answer can be generated automatically during RealTimeValidation
+
+* **Response** Includes all the Answers to questions in the questionnaire from a respondent or an interviewer (associated with a single interview)
+
+   a. **RespondentInterview**
+      - Process where a respondent completes their own interview
+      - Can be incomplete/abandoned
+      - Creates a Response
+   
+   b. **AssistedInterview**
+      - Interview conducted by an interviewer (phone/chat/in-person)
+      - Can be starting fresh or continuing a RespondentInterview (becoming an AssistedInterview): as soon as an interviewer touches the interview, it becomes an AssistedInterview
+
+    Respondent and assisted Response objects are identical in the flow process and are both called Response.
+   
+* **RealTimeValidation** (shorthand: **Validation**)
+   - Frontend/backend validation during interview
+   - Data type checking
+   - Answers coherence verification
+   - Applies to both interview types and to correction process (see below)
+
+## Post-Interview Processing
+
+1. **Audit** (automated) Backend process analyzing Responses (both respondent and assisted)
+
+    * **AuditCheck** A single validation/check. Example: check if household size is consistent with the number of Person objects in the Response. Each AuditCheck as a type: error (which should be reviewed and eventually corrected or rejected), warning (which may be corrected but could be ignored, or could even result in a rejection in some cases), or note (just a message to classify the interview like "has at least one transit trip", "is a single person interview")
+
+    * **AuditCheckResult** The result of an audit check on a specific Response or survey object in a Response (Household, Person, VisitedPlace, Trip, etc.). Example: If the household size does not equal the number of Person objects in the Response, the AuditCheckResult will be false
```

Comment:
Est-ce que ce sera juste `false` le résultat, ou c'est plutôt un flag avec un level (error, warning, note)

Reply:
the level is included in the AuditCheck, so we only need a boolean.